pkg/apis/serving/v1beta1/inference_service_validation.go

```diff
@@ -157,5 +184,71 @@ func validateAutoscalerTargetUtilizationPercentage(isvc *InferenceService) error
 			}
 		}
 	}
+
+	return nil
+}
+
+func validateScalingHPACompExtension(compExtSpec *ComponentExtensionSpec) error {
```

Comment:
Can you help add unit tests for these validations?

Reply:
Sure, adding it.